docs/cybersecurity/ddos/source.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@rspurgeon can you please align the `:` vertically (as done in `source.sql`)?

Reply:
👍  fixed in latest